translator/trace/jaeger/jaegerthrift_to_traces.go

```diff
@@ -64,10 +64,10 @@ func jThriftProcessToInternalResource(process *jaeger.Process, dest pdata.Resour
 
 	attrs := dest.Attributes()
 	if serviceName != "" {
-		attrs.InitEmptyWithCapacity(len(tags) + 1)
+		attrs.EnsureCapacity(len(tags) + 1)
 		attrs.UpsertString(conventions.AttributeServiceName, serviceName)
 	} else {
-		attrs.InitEmptyWithCapacity(len(tags))
+		attrs.EnsureCapacity(len(tags))
```

Comment:
And here.

Reply:
Not a problem, the output param is empty.